### PR TITLE
[PRD-5292] Fix Report Wizard Template selection Bug

### PR DESCRIPTION
**Pull request by Pancho7**
@pentaho/millenniumfalcon

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 4s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl designer/wizard-xul
```
---
:ok_hand: All tests passed!
 
Tests run: **19**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/reporting/job/PR-1309/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 40s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl designer/wizard-xul
```
---
:ok_hand: All tests passed!
 
Tests run: **19**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/reporting/job/PR-1309/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: designer/wizard-xul/src/main/java/org/pentaho/reporting/engine/classic/wizard/ui/xul/steps/LookAndFeelStep.java [hunk 1]
```diff
@@ -582,7 +583,10 @@ public Integer getSelectedTemplate() {
     return selectedTemplate;
   }
 
-  public void setSelectedTemplate( final Integer selectedTemplate ) {
+  public void setSelectedTemplate( Integer selectedTemplate ) {
+    if ( selectedTemplate == -1 ) {
```
Comment (dcleao) [inline designer/wizard-xul/src/main/java/org/pentaho/reporting/engine/classic/wizard/ui/xul/steps/LookAndFeelStep.java]: If the problem was the image control not being initially in sync, calling `imageBinding.fireSourceChanged()`, above, should fix the issue.
So why do you need this here?

Comment (Pancho7) [reply designer/wizard-xul/src/main/java/org/pentaho/reporting/engine/classic/wizard/ui/xul/steps/LookAndFeelStep.java]: Done

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 38s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl designer/wizard-xul
```
---
:ok_hand: All tests passed!
 
Tests run: **19**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/reporting/job/PR-1309/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**